config/routes.rb

```diff
@@ -43,9 +43,9 @@
       to: "gradebooks#show",
       as: :gradebook
 
-    get "subjects/:subject_id/:semester/:year/assignments",
-      to: "assignments#index",
-      as: :assignments
+    get "subjects/:subject_id/:semester/:year/gradebook_assignments",
+      to: "gradebook_assignments#index",
+      as: :gradebook_assignments
 
     resources :assignments, only: [] do
```

Comment:
@derekprior, I think you are correct. I went in and changed all the lib files. Things get a bit weird when I rename line 50 to `resources :gradebook_assignments, only: [] do`. But I will take another look.